.github/actions/build-mbed-edge/action.yaml

```diff
@@ -43,34 +27,13 @@ outputs:
 runs:
   using: composite
   steps:
-    # variables='';
-    # for i in $(env | grep '${{ inputs.env_pattern }}' | awk -F '=' '{print $1}'); do
-    #   variables="--env ${i} ${variables}";
-    # done;
     - name: Setup-Build-Files
       shell: bash
-      run: >
+      run: |
+        cd ${{ inputs.context }}
         cp -a .github/edge-config/${{ inputs.configfiles }}/* config
```

Comment:
Improvement idea - let's move the certs to the scripts-internal to reduce visiblity to public.